ports/libsimpleservo/api/src/lib.rs

```diff
@@ -581,11 +584,12 @@ struct ServoEmbedderCallbacks {
     waker: Box<dyn EventLoopWaker>,
     xr_discovery: Option<Box<dyn webxr_api::Discovery>>,
     vr_init: VRInitOptions,
+    gl: Rc<dyn gl::Gl>,
 }
 
 struct ServoWindowCallbacks {
     gl: Rc<dyn gl::Gl>,
-    host_callbacks: Box<dyn HostTrait>,
+    host_callbacks: Rc<Box<dyn HostTrait>>,
```

Comment:
Bit odd that this is an `Rc<Box<HostTrait>>` rather than an `Rc<HostTrait>`.

Reply:
This is a leftover change from @paulrouget's code, i think this can be undone